crates/hdk_derive/tests/hdk_extern.rs

```diff
@@ -1,5 +1,5 @@
 #[test]
 fn hdk_extern_compile_fail() {
     let t = trybuild::TestCases::new();
-    t.compile_fail("tests/tests/hdk_extern/*.rs");
+    t.compile_fail("tests/hdk_extern/*.rs");
```

Comment:
Would it make sense to specify all the files explicitly, rather than glob matching? I assume then `trybuild` would fail when it couldn't find a file, instead of passing because the glob match just returns no files.

Reply:
> I assume then trybuild would fail when it couldn't find a file

Unfortunately it doesn't fail the test, it just prints a warning text. We would need to create manual logic to check if the files are present. This is why this issue had gone unnoticed before.

Reply:
I would like to add that logic then, to avoid this in the future.